### PR TITLE
Remove MO's extra js confirm. `data-confirm` is handled by rails-ujs

### PR DESCRIPTION
**Pull request by nimmolo**
This should be good to deploy after review.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -686,6 +686,13 @@
     t.boolean "blocked", default: false, null: false
   end
 
+  create_table "visual_groups", charset: "utf8mb3", force: :cascade do |t|
```
Comment (mo-nathan) [inline db/schema.rb:689]: I don't think this should be part of this PR.

**Review by mo-nathan — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
-//= require jquery3
+// = require jquery3
```
Comment (mo-nathan) [inline app/assets/javascripts/application.js:1]: Is this change intentional?

**Comment by mo-nathan** [comment]
I was able to confirm that these changes have the desired effect on my branch.  Pushed the changes to the visual-models branch.

File: db/schema.rb (hunk 1 above)
Comment (nimmolo) [reply db/schema.rb:689]: Oh... that must have come in via merging master. I must have misunderstood what you were trying to say in the email thread... probably trying to save me the extra step. 

Removing from schema now.

File: app/assets/javascripts/application.js (hunk 2 above)
Comment (nimmolo) [reply app/assets/javascripts/application.js]: Nope! Thank you

File: db/schema.rb (hunk 1 above)
Comment (nimmolo) [reply db/schema.rb]: @mo-nathan When I run `bin/rails db:migrate" this keeps recreating the table in the schema, even though there is no migration in this branch that creates this table. 

I've forgotten how to resolve this situation! do you know offhand?

Comment (mo-nathan) [reply db/schema.rb]: It means the table exists in your database, but there's no corresponding migration.  The simplest thing to do is just copy db/schema.rb from master onto your branch and commit it.  To get `rails db:migrate` to work correctly, you can either reload the database or you can just go in to the database console and drop the table.  Then you should be able to run the migrations and schema.rb should not change.

Comment (nimmolo) [reply db/schema.rb]: Got it. Doing that now. Thank you!
